docs/content/preview/contribute/core-database/build-from-src-almalinux.md

```diff
@@ -50,6 +50,53 @@ AlmaLinux 8 is the recommended Linux development platform for YugabyteDB.
 
 {{< /note >}}
 
+# TLDR
+
+{{% readfile "includes/tldr.md" %}}
+
+```sh
+# Modify to your preference:
+shellrc=~/.bashrc
+
+sudo dnf update -y
+sudo dnf groupinstall -y 'Development Tools'
+sudo dnf -y install epel-release
+packages=(
+  ccache
+  cmake3
+  gcc-toolset-11
+  gcc-toolset-11-libatomic-devel
+  golang
+  java-1.8.0-openjdk
+  libatomic
+  maven
+  npm
+  patchelf
+  python39
+  rsync
+)
+sudo dnf -y install "${packages[@]}"
+sudo alternatives --set python3 /usr/bin/python3.9
```

Comment:
There is no `python-310` or `python-311` package yet on Almalinux 8 as far as I can tell.  Also, this is based off of the python section below, so changing both copies of this could be considered out of scope of this PR.